### ISSUE TITLE
New Repository for New design For Chat.susi

### ISSUE THREAD

**Issue by Sid911** [fossasia/chat.susi.ai#1787]
### **What is the purpose**

Currently, there are many people iterating new design for the web app and this repo's purpose would be to consolidate those ideas in one place and give a way to other contributors to hop into the design process to make a better and more refined prototype which can be implemented to the real site

### **What are the setback as of today which require a new repo as essential for collaboration?**
- There is no way of seeing other's file and trying to help on something
- No way of knowing, what everyone is working on.
- No way of sharing files other than just mailing every time something changes.

### **Advantages**
- Having a separate place where re-design of the website is done makes it kind of an isolation from the current projects which are being applied every day to the main repo, as it is a long task and will require a good amount iteration will not cause the problem to the main repo.
- A place where designers could share their design views and collaborate on an issue to solve them.
- Easier to collaborate on a task
- At the end, we will have a refined design which could be implemented to the site

**Will this repo be temporary?**
Yes, after the new design language is implemented this could be reused or deleted

**Comment by AakashMallik** [comment]
The title should be "New design mockup propasals" or something like that. This issue shall be a place for discussions. We are not going for a new repo I guess.
 @Sid911

**Comment by akshatnitd** [comment]
Yes, @AakashMallik is right. Please make the changes.

**Comment by Sid911** [comment]
@akshatnitd  Is that fine?

**Comment by AakashMallik** [comment]
> @akshatnitd Is that fine?

Yeah.. go ahead and invite @niteshkumarniranjan toh put up his work here. You can follow along.

**Comment by niteshkumarniranjan** [comment]
@Sid911 @AakashMallik @akshatnitd I had already opened an issue on 24th of October 
https://github.com/fossasia/chat.susi.ai/issues/1641
